is_core/auth_token/middleware.py

```diff
@@ -33,10 +33,12 @@ def process_response(self, request, response):
         # Save the session data and refresh the client cookie.
         # Skip session save for 500 responses, refs #3881.
         if response.status_code != 500 and hasattr(request, 'token') and request.token.is_active:
-            if not request.token.expiration:
```

Comment:
tedy ten kód má být
```

if request.token.expiration:
    max_age, expires = None
else:
    max_age = settings.AUTH_COOKIE_AGE
    expires = cookie_date(time.time() + max_age)
if not getattr(request, 'auth_token_renewal_exempt', False):
    self.update_token_and_cookie(response, max_age, expires)
return response


```
update_token_and_cookie - předpokládám že už si domyslíš je to jen poslání toho cookie a update tokenu